Frontend/src/app/shared/components/home/home.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <div class="intro-section">
+    <img src="assets/intro-mob.png" class="mob" alt="intro" />
+    <img src="assets/intro-desk.png" class="desk" alt="intro" />
```

Comment:
We should be using HTML for all our designs instead of images. It gives a lot more flexibility in the future.

Reply:
You can also make this responsive with just HTML so the website does not have to download 2 images to the user's PC when they visit.

Reply:
Images removed

Reply:
.gitkeep file re-added.